tests/netlify-deploy.ts

```diff
@@ -168,7 +183,7 @@ export class NextDeployInstance extends NextInstance {
         require('console').log(`Logs: ${buildLogsUrl}`)
       }
     } catch (err) {
-      console.error(err)
+      require('console').error(err)
```

Comment:
not related to main change

this error was not being printed out before - there is some weird console patching going on - so adjusting this to be same as other logs in case those might hide some important detail for us that would help debugging